MoltenVK/MoltenVK/GPUObjects/MVKPixelFormats.mm

```diff
@@ -465,6 +465,7 @@
 													MTLPixelFormat mtlFormat,
 													MTLTextureUsage minUsage,
                                                     bool isLinear,
+                                                    bool isMutableFormat,
                                                     bool isExtended) {
```

Comment:
I'm concerned about the number of image-specific customization elements we pass into this function. Continuing this approach will only get worse with `VK_KHR_image_format_list`.

I'm wondering if we need to start thinking about refactoring this function into the basic conversion which would be handled in `MVKPixelFormats`, and a per-Image overlay function, which would tweak the `MTLTextureUsage` results returned by this function, based on the customization needs of the Image.

Let's go with this for now, but I'll look into that when addressing `VK_KHR_image_format_list`.